nvme-print-json.c

```diff
@@ -3072,6 +3072,15 @@ static void json_nvme_id_ctrl_nvm(struct nvme_id_ctrl_nvm *ctrl_nvm)
 	obj_add_uint64(r, "dmsl", le64_to_cpu(ctrl_nvm->dmsl));
 	obj_add_uint(r, "aocs", le16_to_cpu(ctrl_nvm->aocs));
 
+	if (json_print_ops.flags & VERBOSE) {
```

Comment:
I wonder if we should always print everything. For me the verbose flag makes only sense for the stdout output.

Reply:
My point is that the json output could always print everything, no need make it short. Anyway, your change is fine, let's go with it, I ponder a bit if we should just kill the 'verbose' flag in the json output.